content/en/load-balancer.md

```diff
@@ -7,17 +7,21 @@ tags: ["", "", ""]
 
 ## What it is
 
-A load balancer is a method to distribute incoming network traffic across a group of servers in the back-end. 
-The solution can be software or hardware based.
+A load balancer is a tool that efficiently distributes incoming requests among multiple instances of an application. 
+Take a [microservice](/microservices/) architecture for example, where each service can be [scaled horizontally](/horizontal-scaling/). 
+A load balancer sits in front of a scaled microservice and ensures that no one instance gets the bulk of the requests.
+Load balancers can be software or hardware-based.
 
 ## Problem it addresses
 
-This helps solve the issue related to high availability and distributed systems. 
-When working on an application or service that needs to scale to hundreds of thousands of users, 
-one will often need to distribute that application on multiple servers. 
-The load balancer is the "traffic cop" that routes traffic.
+Modern applications and websites serve thousands of thousands of simultaneous requests from end-users. 
+In order to handle all those requests, [horizontal scaling](/horizontal-scaling/) is nearly unavoidable.
+Unfortunately, [horizontal scaling](/horizontal-scaling/) introduces a new level of complexity which is, how to distribute the incoming traffic to all horizontally scaled services. 
+This is where load balancers come in handy.
 
 ## How it helps
 
-A load balancer will act as front-end for network traffic and clients. 
-It often has various methods to check which server(s) is up and has the lowest load to handle the request.
+Load balancers will distribute all the requests among multiple servers dynamically in a way 
```

Comment:
```suggestion
Load balancers dynamically distribute all incoming requests among multiple services, ensuring that no one service gets the bulk of it while others only get a few or none. 
```
Changed server to services since we are talking about modern (microservices-based) applications